code-challenges.test.js

```diff
@@ -49,19 +153,124 @@ const studyMinutesWeek2 = {
 }
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
 
+// Pseudocode:
+  //   input 2: studyMinutesWeek2
+  // output 2: [10, 15, 20, 45, 60, 65, 100]
+
+  describe("objectToArray", () => {
```

Comment:
No need to rewrite all of this, the first test and function should be able to handle both test variables.